test_vm/src/lib.rs

```diff
@@ -900,6 +943,13 @@ impl ExpectInvocation {
                 id, f, invoc.msg.from
             );
         }
+        if let Some(v) = &self.value {
+            assert_eq!(
+                v, &invoc.msg.value,
+                "{} unexpected value: expected:{}was:{} ",
```

Comment:
```suggestion
                "{} unexpected value: expected: {}, was: {} ",
```

Reply:
For consistency I'm going to keep it compact but might change all of these down the line if they get annoying